scripts/docker/buildpack-deps/Dockerfile.emscripten

```
@@ -32,16 +32,19 @@
 # apparently this currently breaks due to conflicting compatibility headers.
 # Using $(em-config CACHE)/sysroot/usr seems to work, though, and still has cmake find the
 # dependencies automatically.
-FROM emscripten/emsdk:2.0.33 AS base
-LABEL version="11"
+FROM emscripten/emsdk:3.1.19 AS base
+LABEL version="12"
 
 ADD emscripten.jam /usr/src
-RUN set -ex; \
-	cd /usr/src; \
-	git clone https://github.com/Z3Prover/z3.git -b z3-4.8.17 --depth 1 ; \
-	cd z3; \
-	mkdir build; \
-	cd build; \
+RUN set -ex && \
+\
+	apt-get update && apt-get install lz4 && \
```

I'd also try to do this:
```suggestion
	apt-get update && apt-get install lz4 --no-install-recommends && \
```
Less stuff gets installed that way and in images this stuff will likely not be used anyway.